SwiftUI_Homework/SwiftUI_Homework/PlayerView.swift

```diff
@@ -0,0 +1,253 @@
+//
+//  PlayerView.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 16.01.2023.
+//
+
+import SwiftUI
+import AVKit
+
+/// Экран "Плеер" c navigationTitle
+struct PlayerView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let playerText = "Music Player"
+    }
+    var body: some View {
+        NavigationView{
+            MusicPlayer().navigationBarTitle(Constants.playerText)
+        }
+    }
+}
+
+/// Экран "Плеер"
+struct MusicPlayer : View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let posterImagesName = ["alisher", "oxxx"]
+        static let nameSongsList = ["The story of Alisher", "Ой, да!"]
+        static let songsList = ["story", "ox"]
+        static let backwardImageName = "backward.fill"
+        static let gobackwardImageName = "gobackward.15"
+        static let pauseImageName = "pause.fill"
+        static let playImageName = "play.fill"
+        static let goforwardImageName = "goforward.15"
+        static let forwardImageName = "forward.fill"
+        static let formatText = "mp3"
+        static let okText = "Ok"
+        static let saveText = "сохранен в папку загрузки"
+        static let emptyText = ""
+        static let arrowImageName = "arrow.up.circle"
+        static let arrowshapeImageName = "arrowshape.turn.up.left"
+        static let maxMusicTimeValue = ["7:17", "2:22"]
+        static let minMusicTimeValue = "00:00"
+        static let shareText = "Поделиться?"
+        static let yesText = "Да"
+        static let zeroValue: Double = 0
+        static let oneValue: Double = 1
+        static let paddingValue: CGFloat = 20
+        static let spacingHValue: CGFloat = UIScreen.main.bounds.width / 5 - 30
+        static let secondValue: Double = 15
+        static let bottomSpaceValue: Double = 5
+        static let widthValue: CGFloat = 200
+        static let heightValue: CGFloat = 200
+    }
+    
+    // MARK: - ObservedObject
+    @ObservedObject private var playerViewModel = PlayerViewModel()
```

Comment:
```suggestion
    @ObservedObject private var playerViewModel = PlayerViewModel()
```
private значит ниже var body